app/lib/features/pins/providers/notifiers/edit_state_notifier.dart

```diff
@@ -41,6 +44,22 @@ class PinEditNotifier extends StateNotifier<PinEditState> {
 
   void setEditMode(bool editMode) => state = state.copyWith(editMode: editMode);
 
+  Future<void> editTitle(BuildContext context) async {
```

Comment:
arguably shouldn't be on the notifier, should it ?!? I prefer the other pattern where this just a function on the widget ... (feels like `BuildContext` are generally not to be used in notifiers ...)

Reply:
Good notice. Updated code on latest commit. :)